article/test/ArticleFeatureTest.scala

```diff
@@ -494,7 +494,6 @@ import collection.JavaConversions._
         findFirst("aside").getAttribute("role") should be("complementary")
         findFirst("header").getAttribute("role") should be("banner")
         findFirst(".l-footer__secondary").getAttribute("role") should be("contentinfo")
-        findFirst("nav").getAttribute("role") should be("navigation")
```

Comment:
Well - we are talking basically about IE8 here. The page is completly broken there already (no styles, no javascript).

IE8:

<img width="863" alt="screen shot 2017-05-25 at 09 55 20" src="https://cloud.githubusercontent.com/assets/2244375/26443429/4b51ad24-4130-11e7-9b6b-8e66ebf8a5b8.png">

`nav` support matrix:

<img width="1118" alt="screen shot 2017-05-25 at 09 53 06" src="https://cloud.githubusercontent.com/assets/2244375/26443389/12476104-4130-11e7-8804-6a1cb50e9481.png">

Adding the `aria-role` for these browser is an enhancement on top. It has nothing to do with the functionality, so I think it's ok to not add it anymore.